ui/jobs/jobs.js

```diff
@@ -1194,6 +1198,7 @@ class Bars {
       this.o.cpBar.centertext = 'maxvalue';
       this.o.cpBar.bg = computeBackgroundColorFrom(this.o.cpBar, 'bar-border-color');
       this.o.cpBar.fg = computeBackgroundColorFrom(this.o.cpBar, 'cp-color');
+      this.o.cpContainer.classList.add('hide');
```

Comment:
I think it would also maybe be better to do this on the top level `#container` rather than just on the cp bar.  If anybody ever adds other crafting UI, then I think that should be hidden as well.  Hiding the top level container would make this more future proof for that case.